Zahlenraten.Intermediate/apps/Zahlenraten.Intermediate/frontend/src/ZahlenratenCustomForm/ZahlenratenCustomForm.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+import { CustomFormProps } from '../DialogRenderer';
+
+import './ZahlenratenCustomForm.css';
+
+export class ZahlenratenCustomForm extends React.Component<CustomFormProps> {
+
+  private userInput: React.RefObject<HTMLInputElement>;
+
+  constructor(props: CustomFormProps) {
+    super(props);
+    this.userInput = React.createRef();
+  }  
+
+  public render(): JSX.Element {
+    return (
+      <div className='test-class'>
+
+        <h1>Du hast noch {this.props.userTask.tokens[0].payload.leftTries} Versuche</h1>
```

Comment:
```suggestion
      <div className='test-class'>
        <h1>Du hast noch {this.props.userTask.tokens[0].payload.leftTries} Versuche</h1>
```

Reply:
You use `this.props.userTask.tokens[0].payload` a lot, you could move it to a variable :)